package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@fandom/jwplayer-fandom",
-	"version": "2.0.0-beta-6",
+	"version": "2.0.0-dpad-528-13",
```

Comment:
I'll change this version to the next before merge. Leaving as is until then as that's likely to change.